File: seed/swift-sdk/exhaustive/Snippets/Example40.swift
import Foundation
import Exhaustive

private func main() async throws {
    let client = ExhaustiveClient(
        baseURL: "https://api.fern.com",
        token: "<token>"
    )

    _ = try await client.endpoints.params.modifyWithInlinePath(
        param: "param",
        request: "string"
    )
}

try await main()

File: seed/java-sdk/exhaustive/forward-compatible-enums/src/main/java/com/snippets/Example40.java
package com.snippets;

import com.seed.exhaustive.SeedExhaustiveClient;
import com.seed.exhaustive.resources.endpoints.params.requests.ModifyResourceAtInlinedPath;

public class Example40 {
    public static void main(String[] args) {
        SeedExhaustiveClient client = SeedExhaustiveClient.builder()
                .token("<token>")
                .url("https://api.fern.com")
                .build();

        client.endpoints()
                .params()
                .modifyWithInlinePath(ModifyResourceAtInlinedPath.builder()
                        .param("param")
                        .body("string")
                        .build());
    }
}
